gp-templates/translation-row-editor-meta-status.php

```diff
@@ -11,20 +11,25 @@
 	<dt><?php _e( 'Status:', 'glotpress' ); ?></dt>
 	<dd>
 		<?php echo display_status( $translation->translation_status ); ?>
-		<?php if ( $translation->translation_status ) : ?>
-			<?php if ( $can_approve_translation ) : ?>
-				<?php if ( 'current' !== $translation->translation_status ) : ?>
-					<button class="button is-small approve" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-current_' . $translation->id ) ); ?>" title="<?php esc_attr_e( 'Approve this translation. Any existing translation will be kept as part of the translation history.', 'glotpress' ); ?>"><strong>+</strong> <span><?php _ex( 'Approve', 'Action', 'glotpress' ); ?></span></button>
-				<?php endif; ?>
-				<?php if ( 'rejected' !== $translation->translation_status ) : ?>
-					<button class="button is-small reject" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-rejected_' . $translation->id ) ); ?>" title="<?php esc_attr_e( 'Reject this translation. The existing translation will be kept as part of the translation history.', 'glotpress' ); ?>"><strong>&minus;</strong> <?php _ex( 'Reject', 'Action', 'glotpress' ); ?></button>
-				<?php endif; ?>
-				<?php if ( 'fuzzy' !== $translation->translation_status ) : ?>
+		<?php if ( ! ( ( 'changes_requested' == $translation->translation_status ) && ( ! apply_filters( 'gp_enable_changes_requested_status', false ) ) ) ) : // todo: delete when we merge the gp-translation-helpers in GlotPress. ?>
```

Comment:
This `if` is confusing. Can we make it easier? Both remove the double negation and parenthesis.

Reply:
Done here https://github.com/GlotPress/GlotPress/pull/1451/commits/8dc5c3249c49e6a232d32d0fdee74d06098a5546.